impedance/models/circuits/elements.py

```diff
@@ -1,6 +1,21 @@
-import numpy as np
 import cmath
 
+import numpy as np
+
+initial_state = globals().copy()
+non_element_functions = ['element_metadata',
+                         'initial_state',
+                         'non_element_functions',
+                         's',
```

Comment:
What do you think of removing `s` and `p` from here and treating them as elements? (for easier importing)

Reply:
I think that's a good idea, I've almost got all of the tests passing using that.  We just have to make exceptions for them in the "expected outputs given an input" test

Reply:
to be clear - not planning on pushing it right away, so keep picking through the code.  I just wanted to make sure there weren't any clear roadblocks